subscribers.js

```diff
@@ -1,7 +1,39 @@
 "use strict";
 
+const db = require("./db");
+
 // Set of all registered emails.
 // TODO: Implement a real persistent storage solution.
-const gEmails = new Set();
+const Subscribers = {
+  async addUser(aEmail) {
+    if (await this.getUser(aEmail)) {
+      console.log("already added");
+      return;
+    }
+
+    await db.doQuery(
+      `INSERT INTO users ( email ) VALUES ( '${aEmail}' );`);
+    console.log("added!");
+  },
+  async deleteUser(aEmail) {
+    let user =  await this.getUser(aEmail);
+    if (!user) {
+      console.log("user doesn't exist");
+      return;
+    }
+
+     await db.doQuery(
+      `DELETE FROM users WHERE id = '${user.id}';`);
+     console.log("deleted!");
+  },
+  async getUser(aEmail) {
+    let rows = await db.doQuery(
+      `SELECT * FROM users WHERE email = '${aEmail}';`);
+    return rows[0];
```

Comment:
Maybe actually check if this exists, and also check number of rows? Currently leaves it to the caller to check whether the user actually existed. We can do this better.